src/base/static/components/organisms/site-header.js

```diff
@@ -223,20 +189,15 @@ const navItemMappings = {
       position={linkProps.position}
       href={linkProps.navBarItem.url}
     >
-      <NavButton
-        variant={linkProps.navBarItem.variant}
-        color={linkProps.navBarItem.color || "tertiary"}
-        onClick={linkProps.onClick}
-      >
+      <NavButton color={"tertiary"} onClick={linkProps.onClick}>
```

Comment:
It doesn't look like our Nav Item Configs define a `color` or `variant` attribute, which is probably for the better. So I removed this unused code.